tools/scripts/brokenLinkChecker/brokenLinkChecker.js

```diff
@@ -3,10 +3,13 @@
 const blc = require('broken-link-checker');
 const https = require('https');
 const url = require('url');
+const YAML = require('yamljs');
 
 const whitelist = require('./whitelist.json').ignore;
 
-const slackUrl = "https://hooks.slack.com/services/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";
+const slackUrl =
+  "https://hooks.slack.com/services/" +
+  YAML.load('../../../globals.yml')['slack_endpoint'];
```

Comment:
It only gets used in JS tools - might live better in tools/scripts than inside the apps folder.